schedulers/terraform/argo-workflow/argo-events-manifests/eventsource-sqs.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: EventSource
+metadata:
+  name: aws-sqs
+  namespace: argo-events
+spec:
+  template:
+    serviceAccountName: event-sa
+  sqs:
+    example:
+      # jsonBody specifies that all event body payload coming from this
+      # source will be JSON
+      jsonBody: true
+      # aws region
+      region: us-east-1
```

Comment:
region is hardcoded. Is there a way we can allow customers to change the region from the TF code?

Reply:
I modified the region and queue name to be as variable.